src/common/options.cc

```diff
@@ -5414,7 +5414,18 @@ std::vector<Option> get_global_options() {
 
     Option("crimson_osd_scheduler_concurrency", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
     .set_default(0)
-    .set_description("The maximum number concurrent IO operations, 0 for unlimited")
+    .set_description("The maximum number concurrent IO operations, 0 for unlimited"),
+
+    // ----------------------------
+    // blk specific options
+    Option("bdev_type", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_description("Explicitly set the device type to select the driver if it's needed")
+    .set_enum_allowed({"aio", "spdk", "pmem", "hm_smr"})
+    .set_long_description("The relationship between the configured value and driver is: "
+        "aio : KernelDevice, "
```

Comment:
the class names are not user friendly, might want to rephrase them like "aio" for using libaio backend, which is the default one. spdk for NVMe devices driven by SPDK, pmem for PMEM devices driven by PMDK, hmsmr for SMR devices.

Reply:
@tchaikov 
Currnetly, the allowed names are "aio" & "spdk" & "pmem" & "hm_smr", do you mean to change "hm_smr" to "hmsmr"?
Note: The long description here shows the info that which BlockDevice driver will be used after setting "bdev_type".

Reply:
i mean the class names like "KernelDevice" does not make sense to end user, unless she/he reads the source code.

Reply:
@tchaikov 
What's about just removing the "set_long_description" here?

Reply:
fine by me. as it's mostly for advanced user anyway, and not likely to be used in short term.

Reply:
I'll remove the "set_long_description" here.